downstream/modules/platform/con-persisting-data-from-auto-runs.adoc

```diff
@@ -5,29 +5,29 @@
 [id="persisting-data-from-auto-runs_{context}"]
 = Persisting data from auto runs
 
-Consider the local Automation Controller filesystem as counterproductive since that ties the data to that host. If you have a multi-node cluster, then you can contact a different host each time causing issues if you are creating workflows that depend upon each other and created directories. For example, if a directory was only created in one node, while another node runs the playbook, the results would be inconsistent.
+Consider the local {ControllerName} filesystem as counterproductive since that ties the data to that host. If you have a multi-node cluster, then you can contact a different host each time causing issues if you are creating workflows that depend upon each other and created directories. For example, if a directory was only created in one node, while another node runs the playbook, the results would be inconsistent.
 
-The solution is to use some form of shared storage solution, like Amazon S3, Gist, or a role to rsync data to your data endpoint. 
+The solution is to use some form of shared storage solution, such as Amazon S3, Gist, or a role to rsync data to your data endpoint. 
 
-The option exists of injecting data or a configuration into a container at runtime. This can be achieved using the automation controller’s isolated jobs path option.
+The option exists of injecting data or a configuration into a container at runtime. This can be achieved by using the {ControllerName}'s isolated jobs path option.
 
-This provides a way to mount directories and files into an execution environment at runtime. This is achieved through the automation mesh, utilizing ansible-runner to inject them into a Podman container to launch the automation. What follows are some of the use cases for using isolated job paths:
+This provides a way to mount directories and files into an {ExecEnvShort} at runtime. This is achieved through the {AutomationMesh}, utilizing ansible-runner to inject them into a Podman container to start the automation. What follows are some of the use cases for using isolated job paths:
```

Comment:
Replace "utilizing" with "using" as it is a word to avoid in SG.

Reply:
Thanks for flagging this. 
It doesn't seem to be catered for in the Vale rules so I'll take a look at that 👍